### PR TITLE
multiLine fix

### PR DESCRIPTION
**Pull request by royibernthal**
Untested but should work, fairly simple changes.

### CODE REVIEW

File: gui/src/2D/controls/multiLine.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class MultiLine extends Control {
 
     /** Function called when a point is updated */
     public onPointUpdate = (): void => {
-        this._markAsDirty();
+        if (this.isVisible) this._markAsDirty();
```
Comment (deltakosh) [inline gui/src/2D/controls/multiLine.ts]: We need to add a MarkasDirty when is Visible will become true again then

Comment (deltakosh) [reply gui/src/2D/controls/multiLine.ts]: Let me fix that one

Comment (royibernthal) [reply gui/src/2D/controls/multiLine.ts]: I'm on it actually :)

Comment (deltakosh) [reply gui/src/2D/controls/multiLine.ts]: I will resolve it at control level

Comment (deltakosh) [reply gui/src/2D/controls/multiLine.ts]: You can remove it, it will be handle for all controls :)

Comment (royibernthal) [reply gui/src/2D/controls/multiLine.ts]: Okay

Comment (royibernthal) [reply gui/src/2D/controls/multiLine.ts]: Good call, it does make sense to handle this case for all controls :)

**Comment by royibernthal** [comment]
Other than what you're doing at control level - it is of course tested now, forgot to mention

**Pull request merged**